encrypted-media/scripts/playback-persistent-license.js

```diff
@@ -0,0 +1,104 @@
+function runTest(config, testname) {
+
+    var testname = config.keysystem + ', successful playback, persistent-license, '
+                                    + /video\/([^;]*)/.exec( config.videoType )[ 1 ]
+                                    + ', set src before setMediaKeys';
+
+    var configuration = {   initDataTypes: [ config.initDataType ],
+                            audioCapabilities: [ { contentType: config.audioType } ],
+                            videoCapabilities: [ { contentType: config.videoType } ],
+                            sessionTypes: [ 'persistent-license' ] };
+
+
+    async_test( function( test ) {
+
+        var _video = config.video,
+            _mediaKeys,
+            _mediaKeySession,
+            _mediaSource,
+            _releaseSequence = false;
+
+        function onFailure(error) {
+            forceTestFailureFromPromise(test, error);
+        }
+
+        function onMessage(event) {
+            assert_equals( event.target, _mediaKeySession );
+            assert_true( event instanceof window.MediaKeyMessageEvent );
+            assert_equals( event.type, 'message');
+
+            if ( !_releaseSequence )
```

Comment:
This is different from how this check was made in the previous file. Is there a reason for that?